docs/philosophy/outcomes/platform_adoption.md

```diff
@@ -48,50 +60,188 @@ Measures a team’s ability to onboard into a new platform, tool or pattern and
 
     Expressed as a _timestamp_
 
-`Adoption end event(aE)`
+_Adoption end events ( $a_{E}$ )_
 
-:   Developer/application/team does something meaningful using the tool
+:   Developer/application/team does something valuable using the tool
 
     For example:
 
-    - Application deployed in X environment
-    - Code committed to X branch
-    - Workspace spun up
+    - Deploys an application to a specific environment
+    - Completes a story
+    - Commits some code to a specific branch
+    - Logs a certain number of hours of active use
 
     Expressed as a _timestamp_
 
 #### Formulas
 
-`aLT = aE - aS`
+_Adoption Lead Time ( $LT$ )_
+
+:   For any individual adoption event, the adoption lead time $a_{LT}$ can be calculated as follows:
+
+    $$
+    LT = a_{E} - a_{S}
+    $$
 
-### :fontawesome-solid-arrow-trend-up: __Adoption rate__
+**_Average Adoption Lead Time ( $\overline{x}LT$ )_**
 
-Measures the ability of the platform or tool to scale to support multiple teams and products
+:   Given a collection of $N$ individual adoption lead times ( $LT_{1}..LT_{N}$ ), the _average adoption lead time_ $\overline{x}LT$ can be calculated as follows:
 
-#### Variables
+    $$
+    \overline{x}LT = \frac{sum(LT_{1}..LT_{N})}{N}
+    $$
 
-`Adoption end event(aE)`
+### :fontawesome-solid-arrow-trend-up: Adoption rate
 
-`Adoptions at Time T1` (At<sub>1</sub>)
+Adoption Rate ($AR$) measures the ability of the platform or tool to scale to support multiple teams and products. 
 
-`Adoptions at Time T2` (At<sub>2</sub>)
+#### Data Points
+
+The following data points must be gathered from the systems we are aiming to measure.
+
+_Adoption end events ( $a_{E}$ )_
+
+:   Developer/application/team does something valuable using the tool
+
+    For example:
+
+    - Deploys an application to a specific environment
+    - Completes a story
+    - Commits some code to a specific branch
+    - Logs a certain number of hours of active use
+
+    Expressed as a _timestamp_
 
 #### Formulas
 
-At<sub>1</sub> = count(aE)[t<sub>1</sub>]
+_Adoptions at Time $t_{1}$ ( $a^{t_{1}}$ )_
 
-At<sub>2</sub> = count(aE)[t<sub>2</sub>]
+:   The number of Adoption end events $a_{E}$ that have taken place as of timestamp $t_{1}$. Calculated as follows:
 
-AR = (At<sub>1</sub> - At<sub>2</sub> - 1) * 100
+    $$
+    a^{t_{1}} = count(aE)[t_{1}]
+    $$
+
+_Adoptions at Time $t_{2}$ ( $a^{t_{2}}$)_
```

Comment:
here is the same definition as before, no need to repeat. 

But below (in Adoption Rate AR), it would be interesting to give a definition alongside the notation (it is 2 t2 or one of them should be t1?)

Reply:
Yeah, the idea here is to capture the number of adoption events and time 1 and time 2 so that we can subtract t1 from t2 to find the number of new events, or "growth"